docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -40,42 +40,313 @@ The `LogPipeline` resource is managed by the `Telemetry Operator`, a typical Kub
 
 The `Telemetry Operator` watches all Log Pipeline resources and related secrets. Whenever the configuration changes, it validates the configuration (with a Validating Webhook) and generates a new configuration for the Fluent Bit Daemon Set, where several Config Maps for the different aspects of the configuration are generated. Furthermore, referenced secrets are copied into one secret that is mounted to the Daemon Set as well.
 
-## Technical Reference
+## LogPipeline Usage
```

Comment:
It is trying to outline the feature set on base of use cases. It is not about the tutorial steps itself. I think it will be bad to explain the feature set each in individual documents. Also jumping all the time feels disturbing to me